etc/kayobe/inventory/group_vars/controllers/firewall

```diff
@@ -0,0 +1,448 @@
+firewallgen_enable_firewall: True
+
+firewallgen_ipv4_input_allow_rules:
```

Comment:
If we find we need to add ports to this list, is there a way we can do that and ensure they don't disappear when the list is regenerated? Perhaps only worth considering if it comes up.